torchbenchmark/canary_models/orca_2/install.py

```diff
@@ -0,0 +1,9 @@
+import subprocess
+import sys
+import os
+from torchbenchmark.util.framework.huggingface.patch_hf import patch_transformers, cache_model
+
+if __name__ == '__main__':
+    patch_transformers()
+    model_name = os.path.basename(os.path.dirname(os.path.abspath(__file__)))
+    cache_model(model_name)
```

Comment:
Try adding an empty line at the end of files. The linter likes it better :)
```suggestion
    cache_model(model_name)

```